Dockerfile

```diff
@@ -1,12 +1,15 @@
 FROM golang:1.20rc2-alpine3.17
 
-LABEL maintainer="manujurado1@correo.gur.es"
+LABEL maintainer="manujurado1@correo.ugr.es" \
+      version="v0.0.5"
 
 WORKDIR /app/test
 
 RUN adduser --disabled-password --uid 1001 test && chown test:test /app/test
 
-COPY . .
+COPY go.mod go.sum Taskfile.yml ./
+
+ADD pkg ./pkg

```

Comment:
¿Por qué utilizas ADD en vez de COPY?

Reply:
Pues lo había puesto para no poner 2 COPY seguidos, pero mirando las buenas prácticas es mejor poner copy que add, así que lo cambiaré.

Reply:
Cambiado!

Reply:
> Pues lo había puesto para no poner 2 COPY seguidos, pero mirando las buenas prácticas es mejor poner copy que add, así que lo cambiaré.

Pero no hagáis trampas en el solitario, hombre...